### ISSUE TITLE
protobuf-net.Grpc.Native and protobuf-net.Grpc for server and client

### ISSUE THREAD

**Issue by SDP190** [protobuf-net/protobuf-net.Grpc#71]
Hi, would like to know if possible to use protobuf-net.Grpc.Native for server, and the dotnet core version for client (protobuf-net.Grpc, Grpc.Net,Client)? 
Are they compatible? And if yes, would it be supported for the long run?

**Comment by duyang76** [comment]
AFAIK, they are compatible (see examples [here](https://github.com/protobuf-net/protobuf-net.Grpc/blob/master/examples/perf/PerfClient/Program.cs)).

**Comment by mgravell** [comment]
Yes, they are entirely compatible. Ultimately, the whole point of gRPC is that one end could be Java and the other golang, so: it is designed to port between implementations.

For support: the two underlying transports are hosted by Google and Microsoft; protobuf-net.Grpc just reshapes the API and sits on top of them, so: as long as Google and Microsoft keep their end, we should be good. The tricker question is TFMs. 472 is quite easy to support compared to, say, net46, so: that should be fine for the foreseeable.
